testing/makefile

```diff
@@ -127,7 +127,7 @@ test_parser:
 
 test_data:
 # Compile testcase model
-	make compile_testcase_model TESTCASE=testcase2
+#	make compile_testcase_model TESTCASE=testcase2
```

Comment:
This should not be commented.

Reply:
This is because testcase 2 can't be compiled with the updated building library. I will try the new branch